samples/suit/smp_transfer/README.rst

```diff
@@ -0,0 +1,394 @@
+:orphan:
+
+.. _nrf54h_suit_sample:
+
+SUIT DFU: Firmware update on the nRF54H20 SoC
+#############################################
+
+.. contents::
+   :local:
+   :depth: 2
+
+The sample demonstrates how to update and boot the nRF54H20 System-on-Chip (SoC) using the Software Update for Internet of Things (SUIT) procedure on the application and radio cores of the SoC.
+The update on the nRF54H20 SoC can be done over Bluetooth® Low Energy or UART.
+
+Requirements
+************
+
+The sample supports the following development kit:
+
+.. table-from-rows:: /includes/sample_board_rows.txt
+   :header: heading
+   :rows: nrf54h20dk_nrf54h20_cpuapp
+
+You need the nRF Device Manager app for SUIT update over Bluetooth Low Energy:
+
+* `nRF Device Manager mobile app for Android`_
+  (The minimum required version is v1.9.)
+
+* `nRF Device Manager mobile app for iOS`_
+  (The minimum required version is v1.5.)
+
+For a SUIT update over UART, you need to install MCUmgr.
+MCUmgr is a tool that can be used to upload SUIT envelopes through the SMP protocol.
+For more information on MCUmgr installation, see :ref:`zephyr:mcu_mgr`.
+
+Overview
+********
+
+The sample uses one of the following methods to perform a firmware update on the nRF54H20 SoC, featuring Nordic Semiconductor’s implementation of the SUIT procedure:
+
+* Bluetooth Low Energy and the nRF Device Manager app
+* UART and MCUmgr
+
+The sample demonstrates how to perform a DFU of the application and radio firmware.
+
+SUIT is the only DFU procedure supported for the nRF54H20 SoCs.
+To read more about the SUIT procedure, see :ref:`ug_nrf54h20_suit_intro`.
+
+|NCS|-specific SUIT limitations
+===============================
+
+The sample and the SUIT DFU procedure currently only support the application core and the radio core firmware updates.
+Additionally, this sample is an early version and is only meant to showcase the start of Nordic Semiconductor’s implementation of the SUIT DFU procedure.
+
+User interface
+**************
+
+LED 1:
+    The number of blinks indicates the application SUIT envelope sequence number.
+    The default is set to ``1`` to blink once, indicating "Version 1".
```

Comment:
LED 1 or LED 0? @tomchy , @FrancescoSer ? See note at the start of https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/nrf/samples/matter/light_bulb/README.html#user-interface

Reply:
Waiting for SME response.

Reply:
I think we can apply here the rules Bartek posted after the meeting with Paal.